test/conformance/tests/test_sriov_operator.go

```diff
@@ -171,6 +171,82 @@ var _ = Describe("[sriov] operator", func() {
 			})
 		})
 
+		Context("LogLevel affects operator's logs", func() {
```

Comment:
do we really want this ? 

or we just want to make sure that Options.Level.(zzap.AtomicLevel).Level() has changed following a change in sriovoperator config ?

Reply:
How can we check that? Do you refer to the unit tests in `pkg/log/log_test.go`? I'm torn because I think this e2e test is likely to become flaky. Too complicated to be robust

Reply:
ah this one is a different binary 🤦 

i dont have a better idea except: "lets not test it"

Reply:
Let me see if I'm able to test it at [`controllers/sriovoperatorconfig_controller_test.go`](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/bf30003a3d9db7431864b7c8b2ad0a4b2335808b/controllers/sriovoperatorconfig_controller_test.go) level. It would be much faster and more robust. Otherwise, I'm ok with dropping this.